### DISCUSSION TITLE
Custom attributes in many to many relationship

### DISCUSSION THREAD

**Discussion by Adambrb** [strapi/strapi#6668] [Legacy Topics]
Hello !
I have 2 collections : "training" and "user" , and many to many relationship between them.
I would like to add score field in join table to classify each user according to the score achieved for a given training, but I don't know how to do that, can you help me please ?

**Comment by gonzarascon** [comment]
Hi! I've had this same issue. 
A workaround I found is to create another content type (say TrainingUser) wich has relations to both Training and User, in that content type you should add the score field. 
Then you should use lifecycle hooks to auto-fill this intermediate model each time a new user (or training) is added.
This certainly doesn't feel clean at all but should do the trick for the moment. 

I'm trying to find a way to hide this intermediate model on the panel so it doesn't confuse the admin user.

**Reply by Adambrb** [reply]
I have looked everywhere but I think there are no other solutions at the moment. Thank you for your reply.
I don't know how to hide a model on the panel sorry.
